### DISCUSSION TITLE
CTkTextbox get input

### DISCUSSION THREAD

**Discussion by omerbm** [TomSchimansky/CustomTkinter#262] [General]
I add the CTkTextbox to my software but I can't find any method to get what the user enter in the textbox
is there any work around some of you use to get the input

**Comment by abhaymakes** [comment]
I think currently there is no function to get the input from CTkTextBox, i'd suggest using tkinter TextBox till `get` function gets added in CustomTkinter.

**Comment by Great-special** [comment]
'CTkTextbox' object has no attribute 'get'.

**Comment by JaiCode08** [comment]
I found a solution from the discussions. Go to this link: https://github.com/TomSchimansky/CustomTkinter/discussions/315. Copy the script. Go into the module files and then the widgets folder. There you will see the textbox widget file, `ctk_textbox.py`. Replace that script with the copied script and you should be able to use the get function.
